### DISCUSSION TITLE
Import additional weight data

### DISCUSSION THREAD

**Discussion by giterest** [tcgoetz/GarminDB#129] [Ideas]
Hi there,

just wanted to ask if there's a way to also import additional weight data (from an Index scale) like body fat, body water %, ...?

Many thanks & KR,
Chris

**Comment by tcgoetz** [comment]
It definitely can be done, but because its from a device that many may not have, I would do it with a plugin. It would require extending the plugin support from just FIT files to JSON files.

If you look in ~/HealthData/weight/weight_*.json you will see the data you are looking for. The plugin support is needed to get the data into the DB.

**Reply by giterest** [reply]
Hi there, yes you're right - everything is in there. Will have a look in you plugins docs and see what I can do..
